Menu/Tab/index.jsx

```diff
@@ -82,6 +84,16 @@ export default React.createClass({
     }
   },
 
+  componentWillUnmount () {
+    window.removeEventListener('resize', this.resizeListener)
+  },
+
+  resizeListener () {
+    window.requestAnimationFrame
+      ? window.requestAnimationFrame(() => update(this))
```

Comment:
Found it! https://github.com/klarna/ui/blob/0ff20fbcce4b6013723f73d9135740ce2ab194e0/lib/debounce.js